sections/data-loading.tex

```diff
@@ -0,0 +1,10 @@
+As the purpose of the project is to create a system in which a user should be able to see the weather forecast for specific area, or more specifically the temperature forcast, we have developed a system that leverages several models to do the temparature forecasting. 
+The goal is to allow the user to interact with the system in such a way that the user can see the accuracy of the different models. 
+The user will be able to select a region for which they want to predict the temperature. The results will be shown both as a graph that will display the predicted value versus the actual value as well as the specific set of temperatures of that region for a given time period into the future. This is similar to what one may see on regular weather forecasts.
+
+In order to achieve this we have constructed a pipeline that preprocesses the data, trains our models, and the resulting models are then used to do the forecasting which is displayed on a web application. 
+Figure xx shows the architecture of the pipeline in broad terms. 
+Note that since the transformer model is the best performing model, which is shown in section (experiment afsnit), this is the model that we chosen to include and describe as part of the pipeline. Regardless of this fact, the other models are used into the pipeline in a similar fashion.
```

Comment:
```suggestion
Note that since the transformer model is the best performing model, which is shown in section (experiment afsnit), this is the model that we chose to include and describe as part of the pipeline. Regardless of this fact, the other models are used in the pipeline in a similar fashion.
```